src/libraries/Microsoft.Extensions.Logging.Console/tests/Microsoft.Extensions.Logging.Console.Tests/ConsoleLoggerTest.cs

```diff
@@ -1107,26 +1107,6 @@ public void WriteCore_NullMessageWithNullException(LogLevel level)
             Assert.Empty(sink.Writes);
         }
 
-        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsThreadingSupported))]
-        public void LogAfterDisposeWritesLog()
```

Comment:
~Why's this test being deleted?~ Nevermind - it moved to ConsoleLoggerProcessorTests.cs.